src/ch/blinkenlights/android/vanilla/CoverCache.java

```diff
@@ -435,7 +432,7 @@ public Bitmap createBitmap(Song song, long maxPxCount) {
 				}
 
 				if (inputStream == null && (CoverCache.mCoverLoadMode & CoverCache.COVER_MODE_SHADOW) != 0) {
-					String shadowPath = "/sdcard/Music/.vanilla/"+(song.artist.replaceAll("/", "_"))+"/"+(song.album.replaceAll("/", "_"))+".jpg";
+					String shadowPath = Environment.getExternalStorageDirectory()+"Music/.vanilla/"+(song.artist.replaceAll("/", "_"))+"/"+(song.album.replaceAll("/", "_"))+".jpg";
```

Comment:
I'd go with `String.format` here: too many concatenations and it's not clear what we'll have in the end.